.vim/ftplugin/c.vim

```diff
@@ -1,4 +1,6 @@
-set shiftwidth=2
-set tabstop=2
-set softtabstop=2
+set shiftwidth=4
+set tabstop=4
+set softtabstop=4
 set expandtab 
+
+set cc=100
```

Comment:
I was using Vera++ checker default, but reading Google C++ Guide Style I'll switch to 80 again: https://google.github.io/styleguide/cppguide.html